### PR TITLE
DOCSP-29126 Add OIDC to settings and authentication connection.

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
## DESCRIPTION
Added OIDC to settings and authentication connection.

## STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-29126/connect/advanced-connection-options/authentication-connection/#oidc

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-29126/settings/settings-reference/#settings

## JIRA
https://jira.mongodb.org/browse/DOCSP-29126

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=648b56d9259a8493e5a69901

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/connect/advanced-connection-options/authentication-connection.txt [hunk 1]
```diff
@@ -82,6 +84,44 @@ Procedure
...
+
+         * - Field
+           - Description
+
+         * - Username
+           - Optional. OpenID Connect username.
+
+         * - Auth Code Flow Redirect URI
+           - Optional. Specify a URI where the identity provider redirects you after authentication.
+             The URI must match the configuration of the Identity Provider used by the server.
```
Comment (addaleax) [inline source/connect/advanced-connection-options/authentication-connection.txt:109]:
```suggestion
             The URI must match the configuration of the Identity Provider.
```

The MongoDB server doesn’t use or is aware of this :)

**Review by addaleax — APPROVED** [review]

File: source/settings/settings-reference.txt [hunk 2]
```diff
@@ -131,6 +131,22 @@ You can configure the following settings on the |compass| interface:
...
        |compass-short|.
 
+   * - Browser command to use for OIDC Authentication
+     - OIDC
+     - Specify the browser that |compass-short| redirects you to when authenticating 
+       with the identity provider.
+
+   * - Show Device Auth Flow Checkbox
+     - OIDC 
+     - Show a checkbox on the connection form to enable the device authentication flow.
```
Comment (sarah-olson-mongodb) [inline source/settings/settings-reference.txt:141]: ([term](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#c))
check box

**Review by sarah-olson-mongodb — APPROVED** [review]
Fabulous work @jocelyn-mendez1. One super minor terminology nit and you're good to go. Keep up the good work.
LGTM

File: source/settings/settings-reference.txt (hunk 2 above)
Comment (jocelyn-mendez1) [reply source/settings/settings-reference.txt]: Done.

**Pull request merged**